protocol/shannon/fullnode_account_fetcher.go

```diff
@@ -43,7 +43,7 @@ type cachingPoktNodeAccountFetcher struct {
 	logger polylog.Logger
 
 	// The underlying account client to delegate to when cache misses occur
-	underlyingAccountClient sdk.PoktNodeAccountFetcher
```

Comment:
Please add a TODO to clearly indicate this should be moved to the Shannon SDK.

Reply:
This file and it's functionality will actually no longer be needed at all. I have added a `TODO_TECHDEBT` to capture removing it.